### DISCUSSION TITLE
Using layout from Nuxt3 and router-view from Vue

### DISCUSSION THREAD

**Discussion by vionijannet** [nuxt/nuxt#19210] [Questions] [answered]
I create a page which using layout and router-view, but it seems the main layout cannot show if using router-view.

[Here is the repository](https://stackblitz.com/edit/nuxt-starter-ce3nhv?file=pages/index.vue)

I want to show the layout but still using router-view.
Any idea how to achieve this? 

Thanks in advance.

**Comment by ndragun92** [comment, marked as answer]
app.vue has to look like this to make it work
```html
<template>
  <NuxtLayout>
    <NuxtPage></NuxtPage>
  </NuxtLayout>
</template>
```

https://stackblitz.com/edit/nuxt-starter-hwfn9i?file=app.vue

**Reply by vionijannet** [reply]
Thanks a lot
